zorya/config.py.example

```diff
@@ -10,3 +10,10 @@ DATABASES = {
         'PORT': 5432,
     }
 }
+
+BASE_DIR = ''
```

Comment:
I would consider renaming this since there is a BASE_DIR constant in the settings.py file already.